### PR TITLE
Updates based on feedback

### PR DESCRIPTION
**Pull request by pamgoodrich**
Clarified that stable IDs are in API and markdown files. Added guidelines for using slugs. Vale linting.

### CODE REVIEW

File: 4.-documentation/stoplight-urls.md [hunk 1]
```diff
@@ -6,12 +6,27 @@ stoplight-id: af3cc200b40c3
...
+
+By default, URLs for published API, models, and articles follow this format:
+
+`/docs/{projectSlug]/{stableID}-{title}`
+
+For example, in this published URL of a Markdown article:
+
+`https://meta.stoplight.io/docs/platform/ZG9jOjMwNDI1OTM1-rule-examples`
+
+* `https://meta.stoplight.io/docs` is the workspace URL or custom domain, along with the folder location of published documentation.
```
Comment (marbemac) [inline 4.-documentation/stoplight-urls.md:24]: `https://meta.stoplight.io/docs` is the workspace URL or custom domain, along with the `/docs` prefix which is present for all project documentation.

(kind of a nit, but the word "folder" kind of confused it a bit for me?)

File: 4.-documentation/stoplight-urls.md [hunk 2]
```diff
@@ -6,12 +6,27 @@ stoplight-id: af3cc200b40c3
...
+* `platform` is the name of the project in slug format.
```
Comment (marbemac) [inline 4.-documentation/stoplight-urls.md:25]: Something along lines of the below maybe?

`platform` is the the project's `slug` value. This value can be changed in your project settings.

File: 4.-documentation/stoplight-urls.md [hunk 3]
```diff
@@ -44,39 +59,36 @@ x-stoplight:
...
-
-For example, in this published URL of a Markdown article:
-
-`https://meta.stoplight.io/docs/platform/ZG9jOjMwNDI1OTM1-rule-examples`
-
-* `https://meta.stoplight.io/docs` is the workspace URL or custom domain, along with the folder location of published documentation.
-* `platform` is the name of the project in slug format.
-* `ZG9jOjMwNDI1OTM1` is the stable ID.
-* `rule-examples` is the title of the article in slug format.
+* For API files, published URLs for endpoints and models have different stable IDs. The root-level stable ID is used to generate the child stable IDs, however.
```
Comment (marbemac) [inline 4.-documentation/stoplight-urls.md:66]: Hmm - how many folks have asked about this specific question. Just not sure if this sentence will confuse more readers than it helps 🤔.

**Review by marbemac — CHANGES_REQUESTED** [review]

File: 4.-documentation/stoplight-urls.md (hunk 3 above)
Comment (pamgoodrich) [reply 4.-documentation/stoplight-urls.md:66]: We've had (I think) 3 customers ask how the API URLs are constructed.

File: 4.-documentation/stoplight-urls.md [hunk 4]
```diff
@@ -44,39 +59,36 @@ x-stoplight:
...
+
 #### API Example
 
+The following example assumes that the API contains:
+
+* These paths: `/users/{userId}` and `/user`
+* These operations: `Get User`, `Update User`, and `Create New User`
+* A User model
+
+The endpoin
```
[the remark this answers is not in the document]
Comment (pamgoodrich) [reply 4.-documentation/stoplight-urls.md:90]: Oops! Thank you.

File: 4.-documentation/stoplight-urls.md (hunk 3 above)
Comment (pamgoodrich) [reply 4.-documentation/stoplight-urls.md]: I removed this sentence:

The root-level stable ID is used to generate the child stable IDs, however.

**Review by marbemac — APPROVED** [review]

**Pull request merged**